libraries/standard/mqtt/cbmc/proofs/MQTT_SerializeUnsubscribe/MQTT_SerializeUnsubscribe_harness.c

```diff
@@ -54,9 +54,9 @@ void harness()
     if( pSubscriptionList != NULL )
     {
         /* The output parameter pPacketSize of the function MQTT_GetConnectPacketSize()
-         * must not be NULL. packetSize returned is not used in this proof, but 
-         * is used normally by the application to verify the size of their 
-         * MQTTFixedBuffer_t. MQTT_SerializeConnect() will use the remainingLength 
+         * must not be NULL. packetSize returned is not used in this proof, but
+         * is used normally by the application to verify the size of their
```

Comment:
grammar: _...to verify the size of __its__ MQTTFixedBuffer_t._

Reply:
Sure to all